.github/workflows/mc-release.yml

```diff
@@ -0,0 +1,146 @@
+name: Build & Release to Maven Central
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      JVM_OPTS: -Xmx4800m
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.1.0
+
+      - name: set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+
+      - name: Restore Dependencies From Cache
+        id: cache-dependencies-restore
+        uses: actions/cache/restore@v4
+        with:
+          path: ~/.gradle/caches
+          key: jars-${{ checksum "build.gradle" }}-gradle-${{ checksum "hyperion-core/build.gradle" }}
+
+      - name: Build Dependencies
+        run: ./gradlew androidDependencies
```

Comment:
Afaik this task only displays dependencies, it doesn't do any building. I suppose it implicitly downloads them as well, which is maybe the goal for the caching?